webapp/channels/src/components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx

```diff
@@ -108,11 +93,7 @@ const EnterpriseEditionLeftPanel = ({
 
     const handleClickAddSeats = () => {
         trackEvent(TELEMETRY_CATEGORIES.SELF_HOSTED_EXPANSION, 'add_seats_clicked');
-        if (!isSelfHostedPurchaseEnabled || !canExpand) {
-            window.open(expandableLink(unsanitizedLicense.Id), '_blank');
-        } else {
-            selfHostedExpansionModal.open();
-        }
+        window.open(expandableLink(unsanitizedLicense.Id), '_blank');
```

Comment:
This opens ` ${cwsUrl}/subscribe/expand?licenseId=${licenseId}`;

Do we want that or should we just open to Contact Sales form?

Reply:
This code path was only reachable via a button that would only appear if self hosted expansion was allowed, so it would have never showed/triggered. I've left the button now and removed the check for self hosted expansion so it'll show for everyone, and direct to contact sales form.